src/components/previewCard/imageHelper.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC } from "react"
+
+import styles from "./index.module.css"
+
+interface Props {
+  imageSrc?: string
+  pictureAlt?: string
+  handleOnErrorImage?: (e) => void
+}
+
+const ImageHelper: FC<Props> = ({ imageSrc, pictureAlt, handleOnErrorImage }) =>
+  imageSrc ? (
+    <img
+      src={imageSrc}
+      alt={pictureAlt}
+      className={
+        pictureAlt == "logo"
```

Comment:
can you try to use a new prop instead of the `pictureAlt` value?

Reply:
@ufkaya something like `isLogo` ?

Reply:
Yes